src/common/Types.ts

```diff
@@ -368,6 +368,7 @@ export interface CommonAccessibilityProps {
 
     // Desktop only.
     tabIndex?: number;
+    valueNow?: number;
```

Comment:
New props should also be exercised (or at least used) in the RXPTest app (in the samples directory). This helps prevent future regressions.

Reply:
Added test to AccessibilityTests to exercise valueNow prop.